jetty-core/jetty-start/src/main/java/org/eclipse/jetty/start/StartArgs.java

```diff
@@ -693,7 +693,12 @@ else if (properties.size() > 0)
                 // TODO module path
 
                 for (Prop property : environment.getProperties())
-                    cmd.addArg(property.key + "=" + property.value);
+                {
+                    String value = property.value;
```

Comment:
Can this ever be a genuine environment variable that should appear unquoted?

Reply:
I don't think so, as the environment should be the properties after expansion.